### ISSUE TITLE
Tailwind breaks rails mailer generator for namespaced files

### ISSUE THREAD

**Issue by trueinviso** [rails/tailwindcss-rails#272]
Rails 7.0.5.1

```
rails new mailer --css tailwind
cd mailer
rails g mailer admin/reservations_report send_csv
```

Generates error:

```
 create  app/mailers/admin/reservations_report_mailer.rb
 invoke  tailwindcss
 create    app/views/admin/reservations_report_mailer
 Could not find "layout.text.erb" in any of your source paths. Your current source paths are: 
 /Users/trueinviso/.rvm/gems/ruby-3.1.1/gems/tailwindcss-rails-2.0.29-arm64-darwin/lib/generators/tailwindcss/mailer/templates
```

Everything works as expected without the namespace, namespacing also works as expected in a new rails app without the tailwindcss-rails gem.

**Comment by freibuis** [comment]
I can confirm this also

work around is to make it with no namespace.. then modifier the file created with namespace details

**Comment by flavorjones** [comment]
Proposed fix at #308
